idem_azurerm/states/azurerm/redis/operations.py

```diff
@@ -284,6 +301,29 @@ async def present(
         )
 
     if "error" not in cache:
+        if wait_for_ready and action == "create":
+            if check_interval < 30 or check_interval > 600:
+                log.error(
+                    "The time interval specified within the check_interval parameter is not within the range of allowed values, so it has been reset to 60 seconds."
+                )
+                check_interval = 60
+            time_passed = 0
+            max_wait_timeout = 3000
```

Comment:
bump this to 3600 to make it an even number and make it very clear in the `wait_for_ready` docs that this could add an hour to your workflows...

Reply:
we could also make this value configurable